questionnaire/questionnaire-service.js

```diff
@@ -588,18 +586,131 @@ function createQuestionnaireService({
         };
     }
 
+    async function createSubmission(questionnaireId) {
+        // 1) get questionnaire instance.
+        const questionnaire = await getQuestionnaire(questionnaireId);
+        if (!questionnaire) {
```

Comment:
`getQuestionnaire` ends up throwing already, not sure this will be hit

Reply:
removed. Hangover from previous iteration